src/index.html

```diff
@@ -1,14 +1,79 @@
-<!DOCTYPE html>
-<html lang="en">
+
+<!doctype html>
+<html lang="en" class="h-100">
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Hugo 0.80.0">
```

Comment:
what is the Hugo generator?

Reply:
https://gohugo.io/

( I still get notifications on this repo lol )

Reply:
thanks lol